scripts/initialize.sh

```diff
@@ -72,9 +76,46 @@ while getopts "n:o:p:d:b:a:s:k:h" opt; do
 done
 shift $(expr $OPTIND - 1) # remove options from positional parameters
 
+CONFIG_DIR=${config_dir:-$CONFIG_DIR_DEFAULT}
+
+# read from the config file if a stake_owner is not provided as parameter
 if [ -z "$stake_owner" ]; then
-   echo 'Stake owner address must be provided. See --help'
-   exit 1
+   printf "\nReading stake owner address from the config file..."
+
+   # read from the config file when the stake owner is not provided
+   config_files=($CONFIG_DIR/*.toml)
+   config_files_count=${#config_files[@]}
+   while :; do
+      printf "\nSelect client config file: \n"
+      i=1
+      for o in "${config_files[@]}"; do
+         echo "$i) ${o##*/}"
+         let i++
+      done
+
+      read reply
+      if [ "$reply" -ge 1 ] && [ "$reply" -le $config_files_count ]; then
+         CONFIG_FILE_PATH=${config_files["$reply" - 1]}
+         break
+      else
+         printf "\nInvalid choice. Please choose an existing option number.\n"
+      fi
+   done
+
+   sed -n -e '/^keyFile -/p' $CONFIG_FILE_PATH
+
+   key_file_str=$(grep "^keyFile" $CONFIG_FILE_PATH)
+   # internal field separator, creates array from key_file separated by '"' sign
+   # array[1] is the key file path
+   IFS='"' read -r -a array <<<"$key_file_str"
+
+   # find address:<address> in the key file
+   address_str=$(grep -Eo '"address":.*?[^\\]"' "${array[1]}")
+   # address_array[3] is the ethereum address
+   IFS='"' read -r -a address_array <<<"$address_str"
+   printf "\nStake owner address: ${address_array[3]} \n"
+
+   stake_owner="${address_array[3]}"
```

Comment:
We should get the address from the key file, as there is no guarantee the file name will contain it. We could use `jq` for that:
```suggestion
   stake_owner=$(jq -jr .address $key_file_path)
```

Reply:
> We should get the address from the key file, as there is no guarantee the file name will contain it. 

Yes, we get the address from the key file, not from the file name.

Reply:
I like `awk` and `jq` and the only reason I didn't use it because they need to be installed separately and I wanted to use "native" methods. However, since we use them in other scripts I guess there's no harm using it here as well. [1265fa5](https://github.com/keep-network/keep-core/pull/3163/commits/1265fa59d426363fee21ebfe1188d5e73bdd43be)